tests/websocket/src/test/scala/CurlWSClientSuite.scala

```diff
@@ -66,4 +66,13 @@ class CurlWSClientSuite extends CatsEffectSuite {
       .assertEquals(Nil)
       .parReplicateA_(4)
   }
+
+  clientFixture.test("error") { client =>
+    client
+      .connectHighLevel(WSRequest(uri""))
+      .use_
+      .attemptNarrow[CurlError]
+      .map(_.isLeft)
+      .assert
```

Comment:
IIRC this is munit-ce way to assert for error.
```suggestion
      .intercept[CurlError]
```